lib/Capabilities.php

```diff
@@ -116,6 +116,7 @@ public function getCapabilities(): array {
 				'recording-v1',
 				'chat-get-context',
 				'single-conversation-status',
+				'chat-keep-notifications'
```

Comment:
Using a comma at the last element of array every when is indented, when add another row don't will touch the previous row.

By example, you didn't touch any row, only added a new row. This is a good point when we need to use `git blame`.
```suggestion
				'chat-keep-notifications',
```